examples/angular/grouping/src/app/app.component.ts

```diff
@@ -39,6 +40,7 @@ export class AppComponent {
     columns: columns,
     state: {
       grouping: this.grouping(),
+      pagination: this.pagination(),
```

Comment:
was this broken? I did this example like the `react` one, which doesn't use controlled pagination

Reply:
Table state should not changed if table data changes? With this PR, table states won't change when table data is reset so I added this resetting of pagination in this example.

Reply:
Following the react approach we should have a specific example for pagination and controlled pagination approach I think. wdyt @KevinVandy?

If you're not controlling pagination, the `onStateChange` which is responsible to update the table signal will be updated. That the reason we can't just return a simple `table` value and we have to wrap the methods / return a signal